app/models/send_sms.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.